web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -1017,13 +1014,20 @@ html {
   }
 }
 
+.harmonizer-and-sidebar {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 100%;
+  flex-grow: 1;
+  overflow: auto;
+}
+
 /* Grid */
 #harmonizer-root {
-  overflow: hidden;
   width: 100%;
-  height: calc(100vh - 362px) !important;
-  float: left;
-  margin-top: 8px;
+  height: 100%;
+  overflow: auto;
```

Comment:
In my opinion removing the `overflow: auto` here makes a for a cleaner scroll
[Screencast from 06-05-2024 04:03:05 PM.webm](https://github.com/microbiomedata/nmdc-server/assets/40494088/634ff2e8-f1c9-44ef-92be-34b407e01c76)

Reply:
Whoa, what OS is that? That looks _way_ worse than on my Mac. The nested scrolling areas is exactly what we're trying to avoid here. I might need to pull out a VM and test on other platforms if that's the case.

Reply:
Sorry I had to look at that video a second time. I got so distracted by the visual scrollbars (and the fact that there were two sets of them) that I think I missed the real point that with my changes the header rows aren't staying visible. That's definitely not right and I'll definitely need to address that before going forward. But was there anything else you meant by "a cleaner scroll"?